functions.php

```diff
@@ -0,0 +1,53 @@
+<?php
+//Create PDO connection including et fetch mode
+/**
+ * Create PDO connection including et fetch mode
+ *
+ * @return PDO a database connection
+ */
+function getDb(){
+    $db = new PDO('mysql:host=db; dbname=diana-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/**
+ * Retrieves the drums from the database
+ *
+ * @param PDO $db database connection
+ *
+ * @return array drums retrieved from the database
+ */
+function retrieveDrums(PDO $db) : array {
+    $query = $db->prepare("SELECT steeldrums.`name`, steeldrums.`manufacturer`, steeldrums.`inches`, steeldrums.`notes`, steeldrums.`color`, steeldrums.`img-url` FROM steeldrums;");
```

Comment:
remove steeldrums from in front of each db field. as there is only 1 table being accessed in this query it doesnt need to be there

Reply:
also the FROM steeldrums should read FROM steel-drums to match the table name